docs/CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ Entries inside each section should be ordered by type:
 
 # 5.1.0 - 2022-12-09
 ## Python API
-* [Changed] Add docstring warnings for metadata ([#3156](https://github.com/quiltdata/quilt/pull/3156))
+* [Added] `dedupe` parameter for `Package.push()` ([#3181](https://github.com/quiltdata/quilt/pull/3181))
```

Comment:
This is a changelog backfill? Belongs in its own PR.

Reply:
This is a result of a recent merge with `master`